### DISCUSSION TITLE
[@types/react-native] missing automaticallyAdjustKeyboardInsets type from 0.69.3

### DISCUSSION THREAD

**Discussion by grgmo** [DefinitelyTyped/DefinitelyTyped#61489] [Issues with a @types package]
[@typescript/react-native]
`automaticallyAdjustKeyboardInsets` seems to have been removed from the `0.69.3` type definitions within `ScrollViewPropsIOS`

[a comment by an automated account is not shown]

**Comment by kelset** [comment]
it looks like the prop is still present in the code: https://github.com/facebook/react-native/blob/0.69-stable/Libraries/Components/ScrollView/ScrollView.js#L180 so yeah, maybe submit a PR to the types to add it?

**Reply by grgmo** [reply]
opened PR here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61506
